docs/getting_started.rst

```diff
@@ -110,111 +117,56 @@ The entries that define the FPGA board parameters have more values than the host
    remote_tmp = /opt/nengo-pynq/params
    udp_port = 0
 
-For whichever board you are using, make sure the lines in the appropriate sections are uncommented (remove the leading # **and** space so it appears as above). These default values should be correct unless you've modified the settings or installation of your FPGA board. These parameters are described here but modifications of these values will be described in the board-specific documentation.
+For whichever board you are using, make sure the lines in the appropriate
+sections are uncommented (remove the leading # *and* space so it
+appears as above). These default values should be correct unless you've
+modified the settings or installation of your FPGA board. These parameters are
+described here but modifications of these values will be described in the
+board-specific documentation.
 
 - **ip**: IP address of the FPGA board.
-- **ssh_port**: The port used to open ``SSH`` communications between the host and FPGA board.
-- **ssh_user**: ``SSH`` username to use to login to the board.
-- **ssh_pwd**: Password for ``ssh_user`` to use to login to the board. Note that the ``fpga_config`` file supports the use of ``SSH`` keys (see :ref:`ssh-key`) as an alternate form of authentication.
-- **remote_script**: The location of the main communication script on theFPGA board.
-- **dna_script**: The location of the script that extracts the unique device identifier (called device DNA)
-- **remote_tmp**: Temporary location used to store data as it is transferred between the host and FPGA board.
-- **udp_port**: The port used for UDP communications between the host and FPGA board.
-
+- **ssh_port**: The port used to open SSH communications between the host
+  and FPGA board.
+- **ssh_user**: SSH username to use to login to the board.
+- **ssh_pwd**: Password for `ssh_user` to use to login to the board. Note
+  that the `fpga_config` file supports the use of SSH keys
+  (see :ref:`ssh-key`) as an alternate form of authentication.
+- **remote_script**: The location of the main communication script on the FPGA
+  board.
+- **dna_script**: The location of the script that extracts the unique device
+  identifier (called Device ID).
+- **remote_tmp**: Temporary location used to store data as it is transferred
+  between the host and FPGA board.
+- **udp_port**: The port used for UDP communications between the host and FPGA
+  board.
 
 .. note::
-   It should be noted that the FPGA board should be configured such that non-root users do not require a password to perform ``sudo`` commands. Refer to the respective FGPA board documentation for instructions on how to do this.
-
+   It should be noted that the FPGA board should be configured such that
```

Comment:
Do we need this note?

Reply:
Yes. If people set up their own boards (which should be possible), they need to do this.